sentences/pt_br/climate_HassClimateSetTemperature.yaml

```
@@ -0,0 +1,8 @@
+language: pt_br
+intents:
+  HassClimateSetTemperature:
+    data:
+      - sentences:
+          - "(põe | ponha) <zona> a <temperatura>"
+          - "(põe | ponha) a temperatura a <temperatura>"
+          - "(põe | ponha) a temperatura <na_zona> a <temperatura>"
```

(personal use) I would like to give more suggestions here:

```suggestion
          - "(mude | muda | mudar | ajustar | ajuste | ajusta | põe | ponha) <zona> (para | a | em) <temperatura>"
          - "(mude | muda | mudar | ajustar | ajuste | ajusta | põe | ponha) a temperatura (para | a | em) <temperatura>"
          - "(mude | muda | mudar | ajustar | ajuste | ajusta | põe | ponha) a temperatura <na_zona> (para | a | em) <temperatura>"
```